Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/caffe_impl.cpp

```diff
@@ -26,7 +26,7 @@ struct CaffeModelImpl : public ModelInterface {
 	const float inScaleFactor = 0.007843f;
 	const float meanVal = 127.5;
 
-	const float confidence_threshold = 0.9f;
+	const float confidence_threshold = 0.7f;
```

Comment:
No problem here but can you change `opencv2\imgproc.hpp` to `opencv2/imgproc.hpp` in line 7?

Reply:
Fixed.